TicketSystem/TicketSystem/Controllers/MessageController.cs

```diff
@@ -6,24 +6,23 @@
 namespace TicketSystem.Controllers
 {
     [ApiController]
-    [Route("[controller]")]
+    [Route("api/[controller]")]
     public class MessageController : ControllerBase
     {
-        private readonly ILogger<MessageController> _logger;
         private readonly ApplicationContext _context;
 
-        public MessageController(ILogger<MessageController> logger)
+        public MessageController()
         {
             _context = new ApplicationContext();
-            _logger = logger;
         }
 
-        [HttpPost(Name = "PostMessage")]
-        public Message Post(Message postMessage)
+        [HttpPost]
+        public async Task<IActionResult> Post(Message message)
```

Comment:
There's no need to use IActionResult as a return type. Use viewModel\model instead.

Reply:
I can return Ok(message) in this method, is it good?